bcycle-backend/Data/BCycleContext.cs

```diff
@@ -20,10 +21,10 @@ public BCycleContext(DbContextOptions<BCycleContext> options) : base(options)
         public DbSet<TripPhoto> TripPhotos { get; set; }
         public DbSet<TripPoint> TripPoints { get; set; }
         public DbSet<User> Users { get; set; }
-
-
+        
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
+            // TODO: can we remove this?
```

Comment:
Those were left in until we decide on the fate of StartPoint/EndPoint. I think the decision was made, so they can be removed.